backend/server/serializers.py

```diff
@@ -17,9 +17,10 @@ class Meta:
         fields = ('name', 'meeting_set')
 
 class CommentSerializer(serializers.ModelSerializer):
+    nickname = serializers.ReadOnlyField(source='writer.nickname')
     class Meta:
         model = Comment
-        fields = ('id', 'date', 'comment_text', 'parent_meeting', 'writer')
+        fields = ('id', 'date', 'comment_text', 'parent_meeting', 'writer', 'nickname')
```

Comment:
What this nickname field stands for?

Reply:
Take a look at line 20, it says source='writer.nickname'. It shows the nickname of the writer.